pkg/report/runs.go

```diff
@@ -0,0 +1,97 @@
+package report
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/openshift/osde2e/pkg/config"
+	"github.com/openshift/osde2e/pkg/testgrid"
+)
+
+const (
+	hiveLogName     = "hive-log.txt"
+	artifactURLTmpl = "https://storage.googleapis.com/%s/%s/%d/%s/%s"
+)
+
+func (r Report) GetRuns(prefix string, earliest time.Time, cfg *config.Config) (runs []Run) {
+	tg, err := testgrid.NewTestGrid(cfg.TestGridBucket, prefix, cfg.TestGridServiceAccount)
+	if err != nil {
+		log.Printf("Failed to setup TestGrid support: %v", err)
+	}
+
+	ctx := context.Background()
+	started, latestBuildNum, err := tg.LatestStarted(ctx)
+	if err != nil {
+		log.Fatalf("Couldn't get latest record: %v", err)
+	}
+
+	// report on each build
+	for i := latestBuildNum; i > 0; i-- {
+		if i != latestBuildNum {
+			if started, err = tg.Started(ctx, i); err != nil {
+				log.Printf("Error getting started for %d: %v", i, err)
```

Comment:
nit: `started for build %d` (it says it in error output, so minor nit)

Reply:
ack